### PR TITLE
add GetLmicVersion up/down command

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/commands/uplink/GetLmicVersion.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    version: number
+}
+
+
+const COMMAND_ID = 0x021f;
+const COMMAND_TITLE = 'GET_LMIC_VERSION';
+const COMMAND_BODY_SIZE = 2;
+
+const lmicStatusBitMask = {
+    multicastSupport: 2 ** 0,
```
Comment (DarkPark) [inline src/commands/uplink/GetLmicVersion.ts:28]:
```suggestion
    isMulticastSupported: 2 ** 0,
```
?

File: src/commands/uplink/GetLmicVersion.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    fragmentedDataSupport: 2 ** 1
```
Comment (DarkPark) [inline src/commands/uplink/GetLmicVersion.ts:29]:
```suggestion
    isFragmentedDataSupported: 2 ** 1,
    isSmartHomeSupported: 2 ** 2
```
?

File: src/commands/uplink/GetLmicVersion.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    status: ILmicStatus,
+    version: number
+}
+
+
+const COMMAND_ID = 0x021f;
+const COMMAND_TITLE = 'GET_LMIC_VERSION';
+const COMMAND_BODY_SIZE = 2;
+
+const lmicStatusBitMask = {
```
Comment (DarkPark) [inline src/commands/uplink/GetLmicVersion.ts:27]: is it really `status`?

in the NEW_STATUS command they call it `type`

File: src/commands/uplink/GetLmicVersion.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/commands/uplink/GetLmicVersion.ts:29]: isSmartHomeSupported - dead option

File: src/commands/uplink/GetLmicVersion.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/commands/uplink/GetLmicVersion.ts:27]: status renamed to capabilities

File: src/commands/uplink/GetLmicVersion.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/commands/uplink/GetLmicVersion.ts]: fixed

File: src/commands/uplink/GetLmicVersion.ts (hunk 3 above)
Comment (DarkPark) [reply src/commands/uplink/GetLmicVersion.ts:27]: lmicStatusBitMask

**Review by DarkPark — APPROVED** [review]

**Pull request merged**